doc/DataSlice.md

```diff
@@ -18,7 +18,7 @@ Below is the metadata list and a note explaining how they are calculated.
     * `total_rows` (`int`): total rows returned by a user query.
     * `total_returned_rows` (`int`): total of rows returned in a slice of results.
     * `execution_time` (`float`): in milliseconds, query execution time.
```

Comment:
Maybe we should define what a query is in this context? It's important to know what this duration will mean